src/TimeModeling/judiVector.jl

```diff
@@ -351,29 +351,60 @@ end
 
 -(a::Number, b::judiVector{avDT}) where avDT = -1f0 * (b  - a)
 
+# lmul!(number, judiVector)
+function lmul!(b::Number, a::judiVector{avDT}) where avDT
+    typeof(a.data[1]) == SeisCon && throw("Multiplication for OOC judiVectors not supported.")
+    lmul!(b, a.data)
+    return a
+end
+
+# rmul!(judiVector, number)
+function rmul!(a::judiVector{avDT}, b::Number) where avDT
+    typeof(a.data[1]) == SeisCon && throw("Multiplication for OOC judiVectors not supported.")
+    rmul!(a.data, b)
+    return a
+end
+
 # *(judiVector, number)
 function *(a::judiVector{avDT}, b::Number) where avDT
     typeof(a.data[1]) == SeisCon && throw("Multiplication for OOC judiVectors not supported.")
     c = deepcopy(a)
-    for j=1:c.nsrc
-        c.data[j] = c.data[j] .* b
-    end
+    rmul!(c.data, b)
     return c
 end
 
 *(a::Number, b::judiVector{avDT}) where avDT = b * a
 
+# ldiv!(number, judiVector)
+function ldiv!(b::Number, a::judiVector{avDT}) where avDT
+    typeof(a.data[1]) == SeisCon && throw("Division for OOC judiVectors not supported.")
+    if iszero(b)
+        error("Division by zero")
```

Comment:
@throw ....

Reply:
Actually, always attach an Error type to a `throw` same for OOC throws

Reply:
https://docs.julialang.org/en/v1/manual/control-flow/

There is the list of built-ins exceptions there